docs/install/operator/configuring-with-operator.md

```diff
@@ -0,0 +1,50 @@
+# Configuring Knative by using the Operator
+
+The Operator manages the configuration of a Knative installation, including propagating values from the `KnativeServing` and `KnativeEventing` custom resources to system [ConfigMaps](https://kubernetes.io/docs/concepts/configuration/configmap/).
+
+Any updates to ConfigMaps which are applied manually are overwritten by the Operator. However, modifying the Knative custom resources allows you to set values for these ConfigMaps.
+
+Knative has multiple ConfigMaps that are named with the prefix `config-`.
+
+
+All Knative ConfigMaps are created in the same namespace as the custom resource that they apply to. For example, if the `KnativeServing` custom resource is created in the `knative-serving` namespace, all Knative Serving ConfigMaps are also created in this namespace.
+
+The `spec.config` in the Knative custom resources have one `<name>` entry for each ConfigMap, named `config-<name>`, with a value which is be used for the ConfigMap `data`.
+
+## Examples
+
+You can specify that the `KnativeServing` custom resource uses the `config-domain` ConfigMap  as follows:
```

Comment:
Might be overkill, so feel free to reject (LGTM otherwise)

```suggestion
You can specify that the `KnativeServing` custom resource uses the `config-domain` ConfigMap by setting `spec.config.domain` as follows:
```

Reply:
I think it's OK without this but it's up to you :)

Reply:
I'm happy to defer :) 
/lgtm